biome.jsonc

```diff
@@ -26,7 +23,6 @@
     "rules": {
       "recommended": true,
       "correctness": {
-        "noUnusedVariables": "error",
         "useImportExtensions": {
```

Comment:
should this be upstreamed? I would assume so? we're not in a ts-only world yet.

Reply:
I would like to keep this here as we are slowly moving packages to `ts` imports. So it's upto each package to consider this or not.